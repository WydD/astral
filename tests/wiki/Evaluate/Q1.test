<?xml version="1.0" encoding="UTF-8"?>
<astral xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://astral.ligforge.imag.fr/schema/query.xsd">
<query t0="auto">
<evaluate expression="temp+temp2" to="sum">
<source attributes="id,T,temp2,PHYID,temp" id="R" sourceAttributes="id,T,temp2,PHYID,temp" type="Relation"/>
</evaluate>
</query>
<sources>
<source attributes="id,T,temp2,PHYID,temp" file="tests/wiki/Evaluate/R.R" id="R" impl="FileRelationImpl" type="Relation"/>
</sources>
</astral>
