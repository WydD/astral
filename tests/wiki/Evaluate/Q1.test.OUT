id,temp,temp2,T,sum
t=1000
1,27,20,1000,47.0
1,27,20,1000,47.0
1,29,20,1000,49.0
t=1005
1,33,21,1005,54.0
t=1010
1,25,25,1010,50.0
1,31,19,1010,50.0
1,33,17,1010,50.0
